docs/source/conceptual_guides/adapter.md

```diff
@@ -117,4 +117,13 @@ To avoid adding noise to the tokens, the adapter uses zero-initialized attention
 
-The higher `r`, the more trainable parameters, resulting in a larger model capacity and better performance. Besides, due to the chain structure, the orthogonality of HR planes impacts the capacity and regularity of HRA. To achieve a trade-off between the model capacity and regularity, an orthogonality regularizer of the HR planes is added to the loss function. The weight \\(\lambda\\) can control the strength of the regularizer. 
\ No newline at end of file
+The higher `r`, the more trainable parameters, resulting in a larger model capacity and better performance. Besides, due to the chain structure, the orthogonality of HR planes impacts the capacity and regularity of HRA. To achieve a trade-off between the model capacity and regularity, an orthogonality regularizer of the HR planes is added to the loss function. The weight \\(\lambda\\) can control the strength of the regularizer. 
+
+## Bone
+[Bone](https://huggingface.co/papers/2409.15371) is a new PEFT technology different from the LoRA series, reducing training resources and requiring no complex initialization.Bone not only enhances the utilization of original weight information but also emphasizes the internal connections between weights, leading to faster convergence and better data fitting.
+
+<small><a href="https://huggingface.co/papers/2409.15371">Bone: Block Affine Transformation as Parameter Efficient Fine-tuning Methods for Large Language Models</a></small>
+
+Bone reduces the number of trainable parameters by using a block grouping method, and the block computation of weight W effectively promotes information exchange in the original weights, enhancing data fitting capability during fine-tuning. The experiment mentions controlling the size of trainable parameters through b (block size), similar to r (rank) in LoRA. For consistency within PEFT, we also name b as r. Note: Bone's r (b) is special and requires that weight W satisfies the conditions in_features//r==0 and out_features//r==0. Additionally, when in_features == out_features and Bone-r == LoRA-r, Bone's number of trainable parameters is only half that of LoRA.
+
+From the experiments in the paper, it is evident that Bone, with only half the parameters of LoRA, can outperform LoRA variants across various metrics. However, Bone currently has some issues: it is slower than LoRA and requires checkpointing to address excessive memory usage due to intermediate values, which further reduces training speed. But don't worry—if you are training larger models and fully utilizing FLOPs, Bone's speed will not differ much from LoRA's. We can address Bone's issues by implementing a flash-bone operator. If you have the capability or interest, you are welcome to contribute to improving flash-bone!
```

Comment:
Regarding this part:

> But don't worry—if you are training larger models and fully utilizing FLOPs, Bone's speed will not differ much from LoRA's. We can address Bone's issues by implementing a flash-bone operator. If you have the capability or interest, you are welcome to contribute to improving flash-bone!

I think it's not a good fit for the documentation. Let's replace it with something along the lines of "We plan to address this in the future. Contributions are welcome."